JSTests/test262/expectations.yaml

```diff
@@ -1072,17 +1069,23 @@ test/intl402/DurationFormat/prototype/format/numeric-hour-with-zero-minutes-and-
   default: 'Test262Error: Duration is {"hours":0,"minutes":0,"seconds":1} Expected SameValue(«0, 01», «0:00:01») to be true'
   strict mode: 'Test262Error: Duration is {"hours":0,"minutes":0,"seconds":1} Expected SameValue(«0, 01», «0:00:01») to be true'
 test/intl402/DurationFormat/prototype/format/precision-exact-mathematical-values.js:
-  default: 'Test262Error: Duration is {"seconds":9007199254740991,"milliseconds":9007199254740991,"microseconds":9007199254740991,"nanoseconds":9007199254740991} Expected SameValue(«0:00:9,016,215,470,202,186», «0:00:9,016,215,470,202,185») to be true'
-  strict mode: 'Test262Error: Duration is {"seconds":9007199254740991,"milliseconds":9007199254740991,"microseconds":9007199254740991,"nanoseconds":9007199254740991} Expected SameValue(«0:00:9,016,215,470,202,186», «0:00:9,016,215,470,202,185») to be true'
+  default: 'Test262Error: Duration is {"seconds":10000000,"nanoseconds":1} Expected SameValue(«0:00:10,000,000», «0:00:10,000,000.000000001») to be true'
+  strict mode: 'Test262Error: Duration is {"seconds":10000000,"nanoseconds":1} Expected SameValue(«0:00:10,000,000», «0:00:10,000,000.000000001») to be true'
+test/intl402/DurationFormat/prototype/format/style-digital-en.js:
+  default: 'Test262Error: Assert DurationFormat format output using digital style option Expected SameValue(«1 yr, 2 mths, 3 wks, 3 days, 4:05:06», «1 yr, 2 mths, 3 wks, 3 days, 4:05:06.007008009») to be true'
+  strict mode: 'Test262Error: Assert DurationFormat format output using digital style option Expected SameValue(«1 yr, 2 mths, 3 wks, 3 days, 4:05:06», «1 yr, 2 mths, 3 wks, 3 days, 4:05:06.007008009») to be true'
```

Comment:
I'm slightly confused; this test was passing but it was passing for the wrong reason?

Reply:
Yes, because this patch changed the behavior of the partitionDurationFormatPattern harness (https://github.com/WebKit/WebKit/pull/29128/files#r1616903101). This changed the expected value of this test. But the actual value did not change.
I will fix the actual value in a future PR.